.gitlab-ci/packet.yml

```diff
@@ -106,6 +106,11 @@ packet_opensuse-canal:
 #   extends: .packet
 #   when: on_success
 
+packet_centos7-tls-cipher-suites-check:
```

Comment:
The existing addons CI job would be better suited, we don't have to add a new CI job for every flag

Reply:
@Miouge1 packet_centos7-flannel-containerd-addons-ha is ok?

Reply:
> @Miouge1 packet_centos7-flannel-containerd-addons-ha is ok?

Yup that would do

Reply:
Yep! ❤️

Reply:
@Miouge1 @floryut done.